opensoundscape/annotations.py

```diff
@@ -129,22 +128,18 @@ def from_raven_files(
         Args:
             raven_files: list or iterable of raven .txt file paths (as str or pathlib.Path),
                 or a single file path (str or pathlib.Path). Eg ['path1.txt','path2.txt']
+            annotation_column: string name or integer position of column containing annotations
+                - pass `None` to load the Raven file without explicitly
+                assigning a column as the annotation column. The resulting
+                object's `.df` will have an `annotation` column with nan values!
+                - if a string is passed, the column with this name will be used as the annotations.
+                - if an integer is passed, the column at that position will be used as the annotation column.
+                NOTE: column positions are ordered increasingly starting at 1.
```

Comment:
Even though we used "1-indexing" in the previous implementation, I don't think we should anymore. First column being 0 matches all Python conventions.